src/Ninject.Web.AspNetCore.Test/Unit/TestKernelContext.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Ninject.Web.AspNetCore.Hosting;
+using System;
+
+namespace Ninject.Web.AspNetCore.Test.Unit
+{
+	public class TestKernelContext
+	{
+
+		protected IKernel CreateKernel(IServiceCollection collection, AspNetCoreHostConfiguration configuration = null)
+		{
+			var kernel = new StandardKernel();
+			kernel.Load(typeof(AspNetCoreApplicationPlugin).Assembly);
```

Comment:
The real question here is: why does that work in your scenario? For me, this throws an exception for each test since `Ninject.Web.AspNetCore` is automatically loaded and here, manually loaded as well. Publishing should still keep `Ninject.Web.AspNetCore.dll` in the output directory, so this should also be double-loaded for you?

Reply:
No idea. Potentially because I was in a different path when running it, ninject didn't see it. I have fixed as suggested above.